metal/resource_metal_device.go

```diff
@@ -280,7 +280,7 @@ func resourceMetalDevice() *schema.Resource {
 					s, _ := structure.NormalizeJsonString(v)
```

Comment:
Curious why we removed `NormalizeJsonString` from the data source, but left it here. I don't know why this was happening or would need to change. Just curious.

Reply:
There was an internal check in the provider that failed on the StateFunc because it doesn't have sense in computed attribute:
https://github.com/bflad/tfproviderlint/tree/main/passes/S033

I left it in the Read function because it's better to have the JSON normalized.